docs/static/upgrading.asciidoc

```diff
@@ -79,4 +80,37 @@ then this issue can be ignored.
 Note the Elasticsearch Output Index Template change in the <<breaking-changes>> documentation for further insight into
 this change and how it impacts operations.
 
+[[upgrading-logstash-pqs]]
+=== Upgrading with Persistent Queues
```

Comment:
I'm wondering if this content really belongs here. Isn't this topic meant for users who are upgrading *to* 5.4? Seems a little weird to mention 6.x versions in this context. I think you need to customize this info to the context.

Reply:
I believe the thinking was that somebody planning to upgrade might start with the version they were upgrading from.  We want to get it anywhere anybody who needs to know might find it.

Reply:
The topics in this section describe how to upgrade *to* 5.4 from earlier versions. It seems very odd to put information about upgrading from 5.4 to 6.x here (it's not consistent with how we handle versioned info in other places throughout the library). When users upgrading to 5.4 read this, first they'll be confused about seeing 6.x references, then they might try to do the steps...and I don't think the queue.drain option even exists in 5.3.

Reply:
If the steps are required for upgrading from 5.4. to 5.5 (where the queue.drain option *does* exist), then you should document this in the 5.5 docs, not 5.4.